.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -9,24 +9,20 @@ assignees: ''
 
 **Fleet version**: _(head to the "My account" page in the Fleet UI or run `fleetctl --version`)_
 
-**Operating system**: _(e.g. macOS 11.2.3)_
-
-**Web browser**: _(e.g. Chrome 88.0.4324)_
+**Web browser and operating system**: _(e.g. Chrome 88.0.4324 running on macOS)_
```

Comment:
## Background on this one

I've closed a lot of frontend bugs in my life, and I can count on one hand the number of times I've seen bugs where the operating system version of the user agent makes a difference (and they were back in the day with IE6 and some early versions of Firefox)

OS-version agnostic but OS-specific bugs that occur in the same browser version are slightly less rare, but still somewhat rare.

Also, "Operating system" as a prompt by itself doesn't clarify what we're asking for (e.g. the OS of the Fleet server?  The OS of the involved hosts?  The OS of the server where Fleet is deployed?)

In the Sails framework, we went back and forth with the amount of detail to ask for in bug reports, and cut the template's teeth on [nearly ten thousand bugs](https://github.com/issues?q=is%3Aopen+archived%3Afalse+org%3Abalderdashy+org%3Asailshq+org%3Anode-machine+).  Eventually, we realized that less is more.

<img width="926" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/5fe426a9-ed9c-419e-ba27-472de6ec9be1">


That doesn't mean there isn't value to be gained from requesting additional info.  Just that we should be super thoughtful about what it is, and run scenarios in our minds to find the minimally viable template.

When it was just "expected behavior" versus "actual behavior", I think it was pretty strong.

The addition of a question in the bug template might be helpful, but only if people actually fill it out.  It encourages people to [type more words, not of less](https://fleetdm.com/handbook/company#empathy).  (It also provides an opportunity for contradiction or confusion inside the issue, and discourages people from submitting bugs.)